lib/sdr_client/deposit/create_resource.rb

```diff
@@ -2,16 +2,16 @@
 
 module SdrClient
   module Deposit
-    # Uploads a resource (metadata) to the server
-    class UploadResource
+    # Creates a resource (metadata) in SDR
+    class CreateResource
```

Comment:
It looks like this public interface change will only break H2, so no worries re: major version bump. Not that it matters since sdr-client is still 0.x. :laughing: 

LGTM